docs/Guides/getting-startted/converting-to-mcap.md

```diff
@@ -0,0 +1,25 @@
+# Converting To MCAP
+
+If you already have existing data that is not in the MCAP file format, you may want to **convert this non-MCAP data into MCAP files**.
+
+[Install the mcap CLI tool](https://github.com/foxglove/mcap/tree/main/go/cli/mcap#installing) to start converting your non-MCAP data to MCAP files.
+
+### ROS 1
+
+```
+$ mcap convert ../../testdata/bags/demo.bag demo.mcap
+```
+
+### ROS 2
+
+```
+$ mcap convert multiple_files_1.db3 demo.mcap
```

Comment:
this filepath is from one of the rosbag2 test files (either directly or via some example in the CLI I guess). Probably doesn't make much sense out of context, so might be clearer to just say `mcap convert demo.db3 demo.mcap`. If there's another readme file with the same issue we should change it.